Automattic/pocket-casts-ios
# End of Year: update epilogue design

| 📘 Project: #376 |
|:---:|

Update the epilogue story design.

| iPhone 8 | iPhone 14 | iPhone 14 Pro Max |
| -------- | ---------- | ------------------- |
| ![Simulator Screen Shot - iPhone 8 - 2022-11-15 at 10 03 19](https://user-images.githubusercontent.com/7040243/201927045-ba94dc49-bd05-4d1d-9973-a3ccd8f2e6ff.png) | ![Simulator Screen Shot - iPhone 14 - 2022-11-15 at 09 59 14](https://user-images.githubusercontent.com/7040243/201927102-7e00e975-c258-4654-942c-ac0aa4eb50f9.png) | ![Simulator Screen Shot - iPhone 14 Pro Max - 2022-11-15 at 10 01 30](https://user-images.githubusercontent.com/7040243/201927128-8901f9ff-3ad3-42b5-96bb-6cd6aaa6adc6.png) |

## To test

I recommend using these devices to test, given they provide different screen sizes:

2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. Check your stories
4. Navigate to the last story
4. ✅ Check the design
4. Tap "Replay"
4. ✅ Stories should replay

## Checklist
